resources/route53_hosted_zones.go

```diff
@@ -0,0 +1,388 @@
+						Resolver: schema.PathResolver("Id"),
+					},
+					{
+						Name: "message",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "name",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "state",
+						Type: schema.TypeString,
+					},
+					{
+						Name:     "ttl",
+						Type:     schema.TypeBigInt,
+						Resolver: schema.PathResolver("TTL"),
+					},
+					{
+						Name: "traffic_policy_id",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "traffic_policy_type",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "traffic_policy_version",
+						Type: schema.TypeInt,
+					},
+				},
+			},
+			{
+				Name:     "aws_route53_hosted_zone_vpc_association_authorizations",
+				Resolver: fetchRoute53HostedZoneVpcAssociationAuthorizations,
+				Columns: []schema.Column{
+					{
+						Name:     "hosted_zone_id",
+						Type:     schema.TypeUUID,
+						Resolver: schema.ParentIdResolver,
+					},
+					{
+						Name:     "vpc_id",
+						Type:     schema.TypeString,
+						Resolver: schema.PathResolver("VPCId"),
+					},
+					{
+						Name:     "vpc_region",
+						Type:     schema.TypeString,
+						Resolver: schema.PathResolver("VPCRegion"),
+					},
+				},
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+func fetchRoute53HostedZones(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
+	var config route53.ListHostedZonesInput
+	c := meta.(*client.Client)
+	svc := c.Services().Route53
+
+	for {
+		response, err := svc.ListHostedZones(ctx, &config, func(o *route53.Options) {})
+		if err != nil {
+			return err
+		}
+		res <- response.HostedZones
+		if aws.ToString(response.Marker) == "" {
+			break
+		}
+		config.Marker = response.Marker
+	}
+	return nil
+}
+func resolveRoute53hostedZoneTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	r := resource.Item.(types.HostedZone)
+	svc := meta.(*client.Client).Services().Route53
+	resourceId := strings.Replace(*r.Id, fmt.Sprintf("/%s/", types.TagResourceTypeHostedzone), "", 1)
+	tagsOutput, err := svc.ListTagsForResource(ctx, &route53.ListTagsForResourceInput{ResourceId: &resourceId, ResourceType: types.TagResourceTypeHostedzone}, func(options *route53.Options) {})
+	if err != nil {
+		return err
+	}
+
+	if tagsOutput.ResourceTagSet == nil {
+		return nil
+	}
+
+	tags := map[string]*string{}
+	for _, t := range tagsOutput.ResourceTagSet.Tags {
+		tags[*t.Key] = t.Value
+	}
+	resource.Set(c.Name, tags)
+	return nil
+}
+func resolveRoute53hostedZoneDelegationSetID(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
You already get this as part of the response, try creating a new struct and embedding HostedZone inside with delegation_set_id and sending that to res channel. I offered the same thing for the tags situation.